larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -80,6 +86,7 @@ typedef struct {
   std::vector<std::uint8_t> bitpacked_weights_buffer;
   bool is_weight_bitpacked = false;
 
+  bool initialized = false;
```

Comment:
The op, i.e. whether or not the `Init` function succeeded. What do you think is a good name?
- `init_succeeded`
- `op_initialized`
- `conv_params_initialized`
- `op_ready`
- ?

Reply:
i prefer `conv_params_initialized` however can you just double check how it`s normally done in other tflite ops? its very strange to me that thats the only way though an error during the `init` process. maybe instead of returning an error they just though an `std::exception`?

Reply:
I also find it strange. I think its because for builtin ops, the flatbuffer already contains the correct struct, will all values filled in. Their `Init` functions are always empty. They don't have to parse anything or check for correctness, only custom ops have to do that.

I'll go with `conv_params_initialized`.